lib/builders/media/mediaBuilder.ts

```diff
@@ -35,27 +35,18 @@ export class MediaBuilder {
     return this;
   }
 
-  withContentTypeId(contentTypeId: string) {
-    this.contentTypeId = contentTypeId;
+  withMediaTypeId(mediaTypeId: string) {
+    this.mediaTypeId = mediaTypeId;
     return this;
   }
 
   build() {
-    if (this.id == null) {
-      const crypto = require('crypto');
-      this.id = crypto.randomUUID();
-    }
-
     return {
-      id: this.id,
-      parentId: this.parentId || null,
-      contentTypeId: this.contentTypeId || null,
-      values: this.documentValueBuilder.map((builder) => {
-        return builder.build()
-      }),
-      variants: this.documentVariantBuilder.map((builder) => {
-        return builder.build();
-      })
+      values: this.mediaValueBuilder.map(builder => builder.build()),
+      variants: this.mediaVariantBuilders.map(builder => builder.build()),
+      id: this.id || null,
+      parent: this.parentId ? {id: this.parentId} : null,
+      mediaType: {id: this.mediaTypeId || null}
```

Comment:
```suggestion
      mediaType: {id: this.mediaTypeId}
```
The mediaType should always have an Id